stories/features/shields/story.tsx

```diff
@@ -8,8 +8,9 @@ import { storiesOf } from '@storybook/react'
 import centered from '@storybook/addon-centered/dist'
 
 // Components
-import BraveShields from './panel/index'
+import Shields from './index'
 
 storiesOf('Feature Components/Shields', module)
   .addDecorator(centered)
-  .add('Page', () => <BraveShields />)
+  .add('Enabled', () => <Shields enabled={true} />)
+  .add('Disabled', () => <Shields enabled={false} />)
```

Comment:
Can we just use a knob for `enabled` or is there a reason we have two concepts for this one?

Reply:
no special reason, added the knob and toggle state